tests/05_02_01-last_fork_active_powpeg_redeem_script.js

```diff
@@ -0,0 +1,87 @@
+const chai = require('chai');
+chai.use(require('chai-as-promised'));
+const expect = chai.expect;
+const peglib = require('peglib');
+const bitcoin = peglib.bitcoin;
```

Comment:
I see the `peglib` is still being used here. I would rather add a `publicKeyToCompressed` utility function in `utils.js` file or something like that, so you can get rid of the peglib from here.

Reply:
removing the publicKeyToCompressed as you mentioned is not required for this test, thank you